src/parser/xmlHandler.js

```diff
@@ -42,7 +57,7 @@ class XMLHandler {
     var name;
     let nameSpaceContextCreated = false;
     if (descriptor instanceof AttributeDescriptor) {
-      val = toXmlDateOrTime(descriptor, val);
+      val = toXmlDateOrTime(descriptor, val, this.options);
```

Comment:
non-blocking: Could we pass `this.options.date` instead of the full options object? That way, the scope of what options the function needs to be aware of is clear.

Reply:
Yeah, it's possible. Looks like it'll be better and cleaner so I'll fix it

Reply:
Resolved in next commit: https://github.com/loopbackio/strong-soap/pull/525/commits/51c5f1cebf729fecc17e2659d7fa55f1af2844c5